src/components/pages/Invite/Instructions.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { View, StyleSheet } from "react-native";
+import { Text, Button } from "../../shared/elements";
+
+interface Props {
+  isJointVideo: boolean;
+  onContinue: () => void;
+  onBack: () => void;
+}
+
+export const Instructions: React.StatelessComponent<Props> = props => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.back} onPress={props.onBack}>
+        Back
```

Comment:
Should this be a button? Also, why do we need an extra back button?

Reply:
I just did a bit of text since I felt that a full button would be a bit too heavy. I removed the react-nav header, which is why I added our own back button - the react-nav back button wasn't properly backing out of the in-page flow and I felt that this looked nicer as a mostly full screen page.